<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lucky Dog</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbtStart">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 132, 139); color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtStop">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>450</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 132, 139); color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QLabel" name="labLevel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 132, 139); color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>奖品等级</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtMusic">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>540</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 132, 139); color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labNum">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 132, 139); color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>奖品个数</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbLevel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 132, 139); color: rgb(0, 0, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>特等奖</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>一等奖</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>二等奖</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>三等奖</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>四等奖</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pbtBack">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>450</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 132, 139); color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>回退</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbNum">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 132, 139); color: rgb(0, 0, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labPeople">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 132, 139); color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>剩余人数</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledPeople">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 132, 139); color: rgb(0, 0, 0);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtForth">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>450</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 132, 139); color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>前进</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
